_components/search-bar.md

```diff
@@ -9,11 +9,13 @@ lead: Intro text on what is included in this section and how to use it. No more
   <div class="usa-grid">
     <div class="usa-width-one-half">
       <form class="usa-search usa-search-big">
-        <label for="search-field-big">Search Big</label>
-        <input type="search" id="search-field-big">
-        <button type="submit">
-          <span class="usa-search-submit-text">Search</span>
-        </button>
+        <div role="search">
```

Comment:
@msecret this was added bc we were missing the search role on the search form. We cannot add the role to the form tag itself bc form has its own native semantics. This means we can either add it to the wrapper around or inside of form. The reason I didn't add it to the wrapper around it is bc we have grid classes and thought it would be hard to distinguish bc they may not want to, and shouldn't have to use the grid. See: http://adrianroselli.com/2015/08/where-to-put-your-search-role.html